bedrock/products/templates/products/relay/waitlist/phone.html

```diff
@@ -0,0 +1,13 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "products/relay/includes/waitlist-base.html" %}
+
+{% set product = ftl('waitlist-phone-name') %}
+
+{% block waitlist_heading %}{{ ftl('waitlist-heading-phone') }}{% endblock %}
+{% block waitlist_lede %}{{ ftl('waitlist-lead-phone') }}{% endblock %}
```

Comment:
Standardize spelling of lede/lead please.

Reply:
Oh we can, I just spelled the block name differently :)